components/StyledButton.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+export default function StyledButton({
```

Comment:
might be worth adding a few more of the standard attributes through here associated with a button, e.g. disabled. Alternatively you could get props and spread them e.g.
```
import React, { forwardRef } from 'react'

const Button = forwardRef(
    (
        {
            children,
            type = 'button',
            className,
            variant = 'primary',
            size = 'normal',
            pill,
            disabled = false,
            ...props
        }, ref
    ) => (
        <button
            ref={ref}
            disabled={disabled}
            type={type}
            {...props}
        >
            {children}
        </button>
    ));

export default Button
```